### PR TITLE
hardware checker script

### PR DESCRIPTION
**Pull request by evemorgen**
This PR provides an easy script checking for hardware and software requirements to be satisfied

It looks like this:
```logs
2019-04-21 16:14:45 raspberrypi root[511] INFO Checking normal camera enabled
2019-04-21 16:14:45 raspberrypi root[511] INFO OK
2019-04-21 16:14:45 raspberrypi root[511] INFO Checking normal camera connected
2019-04-21 16:14:51 raspberrypi root[511] INFO OK
2019-04-21 16:14:51 raspberrypi root[511] ERROR disconnected hardware detected:
    BerryGPS-IMU malfunction gyroscope
    BerryGPS-IMU malfunction compass
    BerryGPS-IMU malfunction bme280
```
and so on

### CODE REVIEW

File: code/hwchecker/checker.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            raise Exception("Not connected")
+        assert 434.15 < freq < 434.25
+        return {}
+    except AssertionError:
+        return {"hardware": ["bad frequency on LoRa radio"]}
+    except Exception:
+        return {"hardware": ["probably no connection to LoRa radio over SPI"]}
+
+
+checks = [
```
Comment (SlupikB) [inline code/hwchecker/checker.py]: Can't find pynmea2 check here, it's necessary to efficiently parse GPS data to send it via lora.
Also, I can't find any OLED module being checked here.

Comment (SlupikB) [reply code/hwchecker/checker.py]: And fuel gauge 0x36

Comment (evemorgen) [reply code/hwchecker/checker.py]: OLED is here: https://github.com/evemorgen/PlasticMonkeysCansat/pull/43/files#diff-7c87f18affdb9493c991e4d3f81a3504R56
Added `pynmea2` and `fuel gauge`

**Comment by jjp241** [comment]
We need also to check presence of [this](https://github.com/adafruit/Adafruit_Python_SSD1306.git`) Oled Library

**Comment by evemorgen** [comment]
> We need also to check presence of [this](https://github.com/adafruit/Adafruit_Python_SSD1306.git%60) Oled Library

Oled on oled board is dead and we don't have a replacement. Unless somebody will drop by in monster elektronic it is not necessary :P

**Pull request merged**